src/applications/personalization/profile360/actions/paymentInformation.js

```diff
@@ -26,12 +26,18 @@ export function fetchPaymentInformation() {
 
 export function savePaymentInformation(fields) {
   return async dispatch => {
+    let gaClientId;
+    try {
+      // eslint-disable-next-line no-undef
+      gaClientId = ga.getAll()[0].get('clientId');
```

Comment:
This would actually make sense to me as a function exporting from `platform/monitoring/recordEvent`, since it's done in HCA I think as well and this is code doesn't read especially meaningfully. I think @tlei123 may have created a ticket for that after the original implementation.